C/plugins/storage/sqlitelb/common/connection.cpp

```diff
@@ -587,6 +606,151 @@ Connection::Connection()
 		//Release sqlStmt buffer
 		delete[] sqlReadingsStmt;
 
+		if (initialiseReadings)
```

Comment:
Why are doing this isn't init.sql of sqlitelb storage does this?  I am suggesting the following  thing 

when storage.sh fetches the required storage plugin the storage.cpp should also return readings plugin along with storage plugin name. The script will execute (storage.sh) will execute corresponding shell scripts of the required plugins. 

For example if we give storage plugin sqlite and readings plugin sqlitelb then it should execute the shell script  
 sqlite.sh  and sqlitelb.sh respectively.

Reply:
It doesn't, but it assumes the sqlitelb is being used for both config and readings, that is not the case the is solving. Tis is for when we switch sqlitelb in for just the readings. Ideally we want to run the init_readings.ql, but there are issues with that, so this is a short term fix until we get a chance to rethink the initialisation. The same thing holds true if Postgres is used for readings only.